component/client-handler/pom.xml

```diff
@@ -27,6 +27,144 @@
         <relativePath>../../pom.xml</relativePath>
     </parent>
```

Comment:
I think this should go under a different component. Here what client handlers mean is client-authenticaiton-handlers.  This is not a client authentication handler..

Reply:
This is an already available component. I haven't changed anything. I just moved the dependencies from parent pom's dependency management.